evm-template/runtime/src/lib.rs

```diff
@@ -138,8 +137,68 @@ pub type SignedExtra = (
 );
 
 /// Unchecked extrinsic type as expected by this runtime.
+// pub type UncheckedExtrinsic =
+//     generic::UncheckedExtrinsic<Address, RuntimeCall, Signature, SignedExtra>;
```

Comment:
are these deliberately kept as comments? or just forgot to delete?

Reply:
I just forgot to delete it, my bad

Reply:
Fixed!